docs/atlascli/command/atlas-organizations-invitations-update.txt

```diff
@@ -91,16 +91,6 @@ Inherited Options
      - false
      - Human-readable label that identifies the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings. To learn about profiles for MongoCLI, see https://dochub.mongodb.org/core/atlas-cli-configuration-file.
 
-Output
```

Comment:
seems like some of the changes docs has done on master are lost on this PR, we need to recover those changes here

Reply:
Found the issue and reverted. Should be fine now (good spot, thanks!)